src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -586,7 +586,7 @@ private async Task CreateDeploymentBundle(Orchestrator orchestrator, Recommendat
             {
                 if (_toolInteractiveService.DisableInteractive)
                 {
-                    throw ex;
+                    throw;

```

Comment:
Not sure what changed in my dev environment by these `throw ex` started giving me compiler errors due to warnings be treated as errors.